packages/rn-tester-e2e/README.md

```diff
@@ -0,0 +1,129 @@
+# RNTester E2E folder
+
+In this folder we have a local setup for doing E2E testing over RNTester via the usage of [Appium](https://appium.io/) and [WebDriverIO](https://webdriver.io/) and [Jest](https://jestjs.io/).
+
+- [Setting up](#setting-up)
+  - [(one-off) Setting up Appium](#one-off-setting-up-appium)
+  - [Building RNTester app](#building-rntester-app)
+    - [Building for iOS](#building-for-ios)
+    - [Building for Android](#building-for-android)
+  - [Setting up the RNTester E2E folder](#setting-up-the-rntester-e2e-folder)
+- [Testing the RNTester app E2E](#testing-the-rntester-app-e2e)
+- [Adding new tests (and project structure)](#adding-new-tests-and-project-structure)
+
+## Setting up
+
+### (one-off) Setting up Appium
+
+The first step you need to do is to ensure to install the tooling:
+
+```bash
+npm install appium@2.0.0-beta.71 -g
```

Comment:
Why do we need to do a global install of this dependency, version using the version provided as a dependency?

Reply:
@NickGerleman Yep it's needed to run the instance of appium server.